### ISSUE TITLE
getByText doesn't find nodes that are <input type="submit" value="XXX">

### ISSUE THREAD

**Issue by julienw** [kentcdodds/react-testing-library#248]
I use a code that contains an submit button like this:
```js
        <input
          className="compareHome-submit-button"
          type="submit"
          value="Retrieve profiles"
        />
```

With react-testing-library I want to get it using for example `getByText('Retrieve profiles')`, which I expect to work because the text `'Retrieve profiles'` is actually seen by users.

But this doesn't work and fails with this error:
```
    Unable to find an element with the text: Retrieve profiles. This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.
```

Is this error expected in this case? I don't find an easy way to get to the submit button and have to use a matcher function instead...

**Comment by kentcdodds** [comment]
Interesting. I think that the query should be updated to account for this case.

The change will actually be pretty simple in this file: https://github.com/kentcdodds/dom-testing-library/blob/master/src/get-node-text.js

We'll want a test for this as well. Would you be willing to make that contribution?

**Comment by julienw** [comment]
Yes, I'm interested in fixing this, but realistically I won't be able to get to it before Thursday or Friday. And if you see nothing before Friday it means it won't be before next year :-) (Precising this in case anybody wants to work on it before this).

**Comment by julienw** [comment]
I realize that `<input type="button">` is concerned too.
